EnglishTranslation-master/QUEST.tsv

```diff
@@ -3076,31 +3076,31 @@ QUEST_20151224_003083	For example if you find the legendary fantastic library, h
 QUEST_20151224_003084	Wizards are magicians who are the most devoted to fundamental magic.{nl}Isn't going back to the basics the best way to start exploring stronger types of magic?
 QUEST_20151224_003085	Using strong spells to channel strong magic is standard,{nl}but don't depend on only one kind of magic.{nl}You must learn how to conjugate different branches of it.
 QUEST_20151224_003086	The settlers who came to Orsha feel uneasy at the sight of the tree roots surrounding the walls.{nl}They want to burn them but are afraid that Orsha will burn down with them as well.
-QUEST_20151224_003087	Our fiery magic is what sets Pyromancers apart.{nl}But, if you play with fire you cannot control, you'll be nothing but a pyromaniac.{nl}
+QUEST_20151224_003087	Our fiery magic is what sets Pyromancers apart from the others.{nl}But if you play with fire you can't control, you'll be nothing but a pyromaniac.
 QUEST_20151224_003088	Turning bad situations around is what Cryomancers do.{nl}Our cold composure is the main source of our magical power.
-QUEST_20151224_003089	Cryomancers must remain cold and prudent at all times.{nl}Conceit, carelessness and indolence will be the end of you.
+QUEST_20151224_003089	Cryomancers must remain cold and prudent at all times.{nl}Conceit, carelessness, and indolence will be the end of you.
 QUEST_20151224_003090	There are so many people in Orsha now, you can't tell who's a settler and who's a refugee.{nl}I wonder if it's not too much for our lord to handle...
 QUEST_20151224_003091	Psychokinos' brand of magic is different compared to other types.{nl}Believe in yourself and have confidence with endless limits.{nl}These are the sources of the Psychokinos' magical power.
 QUEST_20151224_003092	Even after Medzio Diena we've had our hands full with all sorts of problems.{nl}And we might just be facing the most important of them all.
 QUEST_20151224_003093	All beings in this world are connected in some way.{nl}We can end all conflicts if we accept each other as one and the same.
 QUEST_20151224_003094	Shapeshifting is a very profound type of magic, but simply learning it will not make you stronger.{nl}If you can't use it in battle, you're better off learning something else.
 QUEST_20151224_003095	For Thaumaturges, it's the battle that matters.{nl}You need to control others' properties and shapes in a way that's favorable to you.
-QUEST_20151224_003096	Orsha is also being plagued with all sorts of accidents, so nothing works out.{nl}What we really need now are talented strong people who can help us.
+QUEST_20151224_003096	Orsha is also being plagued with all sorts of incidents, so nothing's been going as planned.{nl}What we really need now are strong and talented people who can help us.
 QUEST_20151224_003097	We don't focus on a single property.{nl}Us Elementalists combine various different elemental properties in an efficient way.
```

Comment:
How about just taking out "Us" then swapping "We" and "Elementalists"? This puts the subject first and keeps it in line with the other masters mentioning their own class in these dialogues